content/pt-br/docs/reference/access-authn-authz/authorization.md

```diff
@@ -0,0 +1,250 @@
+```
+
+A saída é semelhante a esta:
+
+```
+yes
+```
+
+```shell
+# "can-i create" = "posso criar"
+kubectl auth can-i create deployments --namespace prod
+```
+
+A saída é semelhante a esta:
+
+```
+no
+```
+
+Os administradores podem combinar isso com [personificação de usuário](/pt-br/docs/reference/access-authn-authz/authentication/#personificação-de-usuário)
+para determinar qual ação outros usuários podem executar.
+
+```bash
+# "can-i list" = "posso listar"
+
+kubectl auth can-i list secrets --namespace dev --as dave
+```
+
+A saída é semelhante a esta:
+
+```
+no
+```
+
+Da mesma forma, para verificar se uma ServiceAccount chamada `dev-sa` no Namespace `dev`
+pode listar Pods no namespace `target`:
+
+```bash
+# "can-i list" = "posso listar"
+kubectl auth can-i list pods \
+	--namespace target \
+	--as system:serviceaccount:dev:dev-sa
+```
+
+A saída é semelhante a esta:
+
+```
+yes
+```
+
+`SelfSubjectAccessReview` faz parte do grupo de API `authorization.k8s.io`, que
+expõe a autorização do servidor de API para serviços externos. Outros recursos
+neste grupo inclui:
+
+* `SubjectAccessReview` - Revisão de acesso para qualquer usuário, não apenas o atual. Útil para delegar decisões de autorização para o servidor de API. Por exemplo, o kubelet e extensões de servidores de API utilizam disso para determinar o acesso do usuário às suas próprias APIs.
+
+* `LocalSubjectAccessReview` - Similar a `SubjectAccessReview`, mas restrito a um namespace específico.
+
+* `SelfSubjectRulesReview` - Uma revisão que retorna o conjunto de ações que um usuário pode executar em um namespace. Útil para usuários resumirem rapidamente seu próprio acesso ou para interfaces de usuário mostrarem ações.
+
+Essas APIs podem ser consultadas criando recursos normais do Kubernetes, onde a resposta `status`
+campo do objeto retornado é o resultado da consulta.
+
+```bash
+kubectl create -f - -o yaml << EOF
+apiVersion: authorization.k8s.io/v1
+kind: SelfSubjectAccessReview
+spec:
+  resourceAttributes:
+    group: apps
+    resource: deployments
+    verb: create
+    namespace: dev
+EOF
+```
+
+A `SelfSubjectAccessReview` gerada seria:
+```yaml
+apiVersion: authorization.k8s.io/v1
+kind: SelfSubjectAccessReview
+metadata:
+  creationTimestamp: null
+spec:
+  resourceAttributes:
+    group: apps
+    resource: deployments
+    namespace: dev
+    verb: create
+status:
+  allowed: true
+  denied: false
+```
+
+## Usando flags para seu módulo de autorização
+
+Você deve incluir uma flag em sua política para indicar qual módulo de autorização
+suas políticas incluem:
+
+As seguintes flags podem ser utilizadas:
+
+  * `--authorization-mode=ABAC` O modo de controle de acesso baseado em atributos [Attribute-Based Access Control (ABAC)] permite configurar políticas usando arquivos locais.
+  * `--authorization-mode=RBAC` O modo de controle de acesso baseado em função [Role-based access control (RBAC)] permite que você crie e armazene políticas usando a API do Kubernetes.
```

Comment:
I'm curious about the square brackets here. In the original version, there are no links in these bullet points.